lib/checkother.cpp

```diff
@@ -1462,6 +1462,8 @@ void CheckOther::checkConstPointer()
     for (const Token *tok = mTokenizer->tokens(); tok; tok = tok->next()) {
         if (!tok->variable())
             continue;
+        if (!tok->variable()->isLocal() && !tok->variable()->isArgument())
```

Comment:
I thought it was a good thing to warn about global pointers also.. but yes I guess we need to ensure that all pointer usage is seen then before we write the warning. I would say we can warn for a static variable that is declared in the source file. We can warn for a class member if all class methods are defined.

Reply:
so we can look at that in a different PR. Thanks for fixing false positives! :+1:

Reply:
> so we can look at that in a different PR.

Yea these FNs can be addressed in a later PR.